src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -147,137 +115,6 @@ public void should_return_a_403_if_task_was_not_previously_assigned() {
 
     }
 
-    @Test
-    public void should_return_a_204_when_completing_a_task_by_id_with_restricted_role_assignment() {
```

Comment:
Whilst this is an IA test, should we consider covering restricted role ini our IT for WA?

Reply:
Again, this is covered in cft.query IT tests. Can add into controller IT tests if required.